patches/applyPatches.sh

```diff
@@ -23,6 +23,27 @@ if [ ! -d "$repo_root/device/sony/lilac/patches" ]; then
   showError "Failed to find repository root at $repo_root"
 fi
 
+patch_set=all
+script_name=$(basename "$0")
+for arg in "$@"; do
+    shift
```

Comment:
Wrong order: Only a single shift is required
```suggestion
shift
for arg in "$@"; do
```